build.rs

```diff
@@ -407,4 +433,32 @@ fn build_android() -> bool {
     env::var("CARGO_CFG_TARGET_OS")
             .expect("CARGO_CFG_TARGET_OS not set")
             .eq("android")
+}
+
+fn configure<P>(project_dir: P, args: &[&str])
+where
+    P: AsRef<str>,
+{
+    let project = project_dir.as_ref();
+
+    let prog = "./configure";
+
+    let _ = subproc("chmod", project, &["+x", prog]);
```

Comment:
Isn't a `configure` script already executable, at least for `zlib`? Why would it ever be checked in but not executable?
If this is truly necessary then please use Rust API for changing permissions, instead of shelling out.

Reply:
> Isn't a script already executable, at least for ? Why would it ever be checked in but not executable? If this is truly necessary then please use Rust API for changing permissions, instead of shelling out.`configure``zlib`

Ok, I will use the api to achieve this

Reply:
> Isn't a `configure` script already executable, at least for `zlib`?